taplo.toml

```diff
@@ -0,0 +1,10 @@
+[formatting]
+align_entries     = true
+column_width      = 120
+array_auto_expand = false
```

Comment:
Hmm, maybe it better to set this to `true`.

Reply:
@lemmih personally I'm fine with whatever settings, as long as they are consistent across the repo. I'll set it to `true` then. :)